### ISSUE TITLE
Streamline the user sanitization function.

### ISSUE THREAD

**Issue by FGDumitru** [openeuropa/oe_authentication#152]
On sites with a big number of users (100K+), the user Drush sanitization function from the **oe_authentication_user_fields** module is trying to load the entire list of users in memory before sanitizating them and that, in turn, depending on the PHP CLI ram limits, might trigger an **Out of memory** error.

I would propose to change, in the _oe_authentication_user_fields/src/Commands/sql/UserSanitizeCommand.php_ file, the lines:

```
$users = $this->entityTypeManager->getStorage('user')->loadMultiple();
 foreach ($users as $user) {
```

to a more memory conservative way as follows:

```
    /** @var  \Drupal\Core\Entity\EntityTypeManagerInterface $user_storage */
    $user_storage = $this->entityTypeManager->getStorage('user');

    /** @var int[] $user_ids */
    $user_ids = $user_storage->getQuery()->execute();

    foreach ($user_ids as $uid) {
      /** @var \Drupal\user\Entity\User $user */
      $user = $user_storage->load($uid);
```

Thank you.

**Comment by fbota** [comment]
Thanks @FGDumitru for the info.
Sanitization should be done without loading in memory the full data as object - usually sanitization should be done directly by execution of proper SAL queries in order to avoid both performance and memory issues during sanitization. I will raise this to the OE team for fixing.

**Comment by voidtek** [comment]
https://github.com/openeuropa/oe_authentication/pull/154 PR created.

**Comment by brummbar** [comment]
Issue fixed in #155 .
